connecting-to-testnet/building-binary.md

```diff
@@ -1,66 +1,67 @@
 ---
-order: 1
+order: 2
 ---
 
 # Building and Starting a Full Node
 
-## Requirements
-
-* [Go](https://golang.org/) (version 1.15 or higher)
-* [make](https://en.wikipedia.org/wiki/Make_(software))
-* [jq](https://github.com/stedolan/jq)
-* [git](https://git-scm.com/)
-* [curl](https://github.com/curl/curl)
-
+To build and start a full node, you build the binary, and then initialize and start a node. 
 
 ## Building the Binary
 
-Clone the code repository:
+This testnet repo is `b-harvest/gravity-dex`. This repo provides the functionality of: 
 
-```
-git clone --branch v1.0.1 https://github.com/b-harvest/gravity-dex
-```
+- `cosmos/gaia` the source code behind Cosmos Hub
+- `tendermint/liquidity` the module source code of Gravity DEX 
 
-We're using `b-harvest/gravity-dex`, which is `cosmos/gaia` (the source code behind Cosmos Hub) with the `tendermint/liquidity` module (source code of Gravity DEX) imported.
+1. Clone the code repository:
 
-Install node's binary `gaiad`:
+    ```
+    git clone --branch v1.0.1 https://github.com/b-harvest/gravity-dex
+    ```
 
-```
-cd gravity-dex && make install
-```
+1. Install the node's binary `gaiad`:
+
+    ```
+    cd gravity-dex && make install
+    ```
 
 ## Initializing a Node
 
-Initialize your node (replace `YOUR_MONIKER` with any string that identifies your node):
+To initialize your node:
 
 ```
 gaiad init YOUR_MONIKER --chain-id cosmoshub-testnet
```

Comment:
`--chain-id` option is not really required, should we remove it to make the command simpler?

Reply:
Yes!